atlp-rwanda/e-commerce-lydia-32-bn
# setup docker for nodejs and db

What does this PR do?

1.add configured dockerfile

how should this be manually tested?

Install Docker Desktop if you haven't already
login into docker-desktop
login into terminal
run docker login
Pull the Docker image from the registry:
docker pull rwigara/team-lydia:demo
run docker images to see images available on your local
once you find it Run the Docker container:
docker-compose up
Access the backend application at http://localhost:5000/
Any background context you want to provide?
N/A
Screenshots (if appropriate)
N/A

Feel free to ask if any inconvenience
